scripts/reorder_json_keys.py

```diff
@@ -0,0 +1,32 @@
+import json
+import os
+
+from recipe_scrapers._utils import get_abstract_methods
+
+KEYS = get_abstract_methods()
+
+
+def reorder_json_keys(file_path):
+    with open(file_path, encoding="utf-8") as file:
+        data = json.load(file)
+
+    reordered_data = {key: data[key] for key in KEYS if key in data}
+
+    if list(data.keys()) != KEYS:
+        with open(file_path, "w", encoding="utf-8") as file:
+            json.dump(reordered_data, file, indent=2, ensure_ascii=False)
```

Comment:
I re-ordered some JSON keys, then ran the unit tests.. and they failed, as expected!  And then I ran the script suggested by the failure message, and it worked perfectly too.

The only thing I'd suggest to improve that experience is that it would have been useful if the fixup/reorder script produced some output -- at first it wasn't clear to me whether it had done anything at all.

```suggestion
            print(f"Re-ordering JSON keys for: {file_path}"
            json.dump(reordered_data, file, indent=2, ensure_ascii=False)
```